Source/WebKit/UIProcess/API/Cocoa/_WKElementAction.mm

```diff
@@ -40,7 +40,10 @@
 #import <wtf/text/WTFString.h>
 
 #if HAVE(SAFARI_SERVICES_FRAMEWORK)
-#import <SafariServices/SSReadingList.h>
+@interface SSReadingList : NSObject
```

Comment:
I realize this isn't SPI so the normal approach isn't necessarily the right one, but should this be in a separate header?